api/liboni/oni.h

```diff
@@ -31,7 +31,11 @@ extern "C" {
 #define ONI_DEFAULTWRITEPATH   "\\\\.\\xillybus_data_write_32"
 #define ONI_DEFAULTSIGNALPATH  "\\\\.\\xillybus_signal_8"
 
+#ifdef LIBONI_EXPORTS
 #define ONI_EXPORT __declspec(dllexport)
+#else 
+#define ONI_EXPORT __declspec(dllimport)
+#endif
 #else
 #define ONI_DEFAULTCONFIGPATH  "/dev/xillybus_cmd_32"
```

Comment:
Should these definitions be here anymore. They seem like they are not needed because they are driver specific

Reply:
They shouldn't. I probably forgot remove them.